<?xml version="1.0" encoding="UTF-8"?>
<testng-results skipped="3" failed="1" total="8" passed="4">
  <reporter-output>
  </reporter-output>
  <suite name="Automation Testing" duration-ms="62073" started-at="2018-06-15T21:40:28Z" finished-at="2018-06-15T21:41:30Z">
    <groups>
    </groups>
    <test name="SignUp" duration-ms="62073" started-at="2018-06-15T21:40:28Z" finished-at="2018-06-15T21:41:30Z">
      <class name="TracksPageScripts.Linkedin_SignUp_PageScripts">
        <test-method status="PASS" signature="setUp()[pri:0, instance:TracksPageScripts.Linkedin_SignUp_PageScripts@3e57cd70]" name="setUp" is-config="true" duration-ms="165" started-at="2018-06-15T14:40:28Z" finished-at="2018-06-15T14:40:28Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setUp -->
        <test-method status="PASS" signature="Setup()[pri:0, instance:TracksPageScripts.Linkedin_SignUp_PageScripts@3e57cd70]" name="Setup" is-config="true" duration-ms="5069" started-at="2018-06-15T14:40:28Z" finished-at="2018-06-15T14:40:33Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- Setup -->
        <test-method status="PASS" signature="firstTestCase()[pri:1, instance:TracksPageScripts.Linkedin_SignUp_PageScripts@3e57cd70]" name="firstTestCase" duration-ms="6423" started-at="2018-06-15T14:40:33Z" finished-at="2018-06-15T14:40:40Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- firstTestCase -->
        <test-method status="PASS" signature="takeScreenshort(org.testng.ITestResult)[pri:0, instance:TracksPageScripts.Linkedin_SignUp_PageScripts@3e57cd70]" name="takeScreenshort" is-config="true" duration-ms="887" started-at="2018-06-15T14:40:42Z" finished-at="2018-06-15T14:40:42Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[[TestResult name=firstTestCase status=SUCCESS method=Linkedin_SignUp_PageScripts.firstTestCase()[pri:1, instance:TracksPageScripts.Linkedin_SignUp_PageScripts@3e57cd70] output={null}]]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- takeScreenshort -->
        <test-method status="PASS" signature="secondTestCase()[pri:2, instance:TracksPageScripts.Linkedin_SignUp_PageScripts@3e57cd70]" name="secondTestCase" duration-ms="3454" started-at="2018-06-15T14:40:42Z" finished-at="2018-06-15T14:40:46Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- secondTestCase -->
        <test-method status="PASS" signature="takeScreenshort(org.testng.ITestResult)[pri:0, instance:TracksPageScripts.Linkedin_SignUp_PageScripts@3e57cd70]" name="takeScreenshort" is-config="true" duration-ms="893" started-at="2018-06-15T14:40:48Z" finished-at="2018-06-15T14:40:49Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[[TestResult name=secondTestCase status=SUCCESS method=Linkedin_SignUp_PageScripts.secondTestCase()[pri:2, instance:TracksPageScripts.Linkedin_SignUp_PageScripts@3e57cd70] output={null}]]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- takeScreenshort -->
        <test-method status="PASS" signature="thirdTestCase()[pri:3, instance:TracksPageScripts.Linkedin_SignUp_PageScripts@3e57cd70]" name="thirdTestCase" duration-ms="3596" started-at="2018-06-15T14:40:49Z" finished-at="2018-06-15T14:40:52Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- thirdTestCase -->
        <test-method status="PASS" signature="takeScreenshort(org.testng.ITestResult)[pri:0, instance:TracksPageScripts.Linkedin_SignUp_PageScripts@3e57cd70]" name="takeScreenshort" is-config="true" duration-ms="915" started-at="2018-06-15T14:40:54Z" finished-at="2018-06-15T14:40:55Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[[TestResult name=thirdTestCase status=SUCCESS method=Linkedin_SignUp_PageScripts.thirdTestCase()[pri:3, instance:TracksPageScripts.Linkedin_SignUp_PageScripts@3e57cd70] output={null}]]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- takeScreenshort -->
        <test-method status="PASS" signature="fourthTestCase()[pri:4, instance:TracksPageScripts.Linkedin_SignUp_PageScripts@3e57cd70]" name="fourthTestCase" duration-ms="4493" started-at="2018-06-15T14:40:55Z" finished-at="2018-06-15T14:40:59Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- fourthTestCase -->
        <test-method status="PASS" signature="takeScreenshort(org.testng.ITestResult)[pri:0, instance:TracksPageScripts.Linkedin_SignUp_PageScripts@3e57cd70]" name="takeScreenshort" is-config="true" duration-ms="887" started-at="2018-06-15T14:41:01Z" finished-at="2018-06-15T14:41:02Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[[TestResult name=fourthTestCase status=SUCCESS method=Linkedin_SignUp_PageScripts.fourthTestCase()[pri:4, instance:TracksPageScripts.Linkedin_SignUp_PageScripts@3e57cd70] output={null}]]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- takeScreenshort -->
        <test-method status="SKIP" signature="fifthTestCase()[pri:5, instance:TracksPageScripts.Linkedin_SignUp_PageScripts@3e57cd70]" name="fifthTestCase" duration-ms="3240" started-at="2018-06-15T14:41:02Z" finished-at="2018-06-15T14:41:05Z">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[The following asserts failed:
	expected [Please enter a valid email addressss] but found [Please enter a valid email address or mobile number]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: The following asserts failed:
	expected [Please enter a valid email addressss] but found [Please enter a valid email address or mobile number]
	at org.testng.asserts.SoftAssert.assertAll(SoftAssert.java:43)
	at TracksPageScripts.Linkedin_SignUp_PageScripts.fifthTestCase(Linkedin_SignUp_PageScripts.java:170)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:86)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:643)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:820)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1128)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:129)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:112)
	at org.testng.TestRunner.privateRun(TestRunner.java:782)
	at org.testng.TestRunner.run(TestRunner.java:632)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:366)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:361)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:319)
	at org.testng.SuiteRunner.run(SuiteRunner.java:268)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1244)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1169)
	at org.testng.TestNG.run(TestNG.java:1064)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:114)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- fifthTestCase -->
        <test-method status="PASS" signature="takeScreenshort(org.testng.ITestResult)[pri:0, instance:TracksPageScripts.Linkedin_SignUp_PageScripts@3e57cd70]" name="takeScreenshort" is-config="true" duration-ms="885" started-at="2018-06-15T14:41:07Z" finished-at="2018-06-15T14:41:08Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[[TestResult name=fifthTestCase status=SKIP method=Linkedin_SignUp_PageScripts.fifthTestCase()[pri:5, instance:TracksPageScripts.Linkedin_SignUp_PageScripts@3e57cd70] output={null}]]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- takeScreenshort -->
        <test-method status="SKIP" signature="fifthTestCase()[pri:5, instance:TracksPageScripts.Linkedin_SignUp_PageScripts@3e57cd70]" name="fifthTestCase" duration-ms="4336" started-at="2018-06-15T14:41:08Z" finished-at="2018-06-15T14:41:12Z">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[The following asserts failed:
	expected [Please enter a valid email addressss] but found [Please enter a valid email address or mobile number]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: The following asserts failed:
	expected [Please enter a valid email addressss] but found [Please enter a valid email address or mobile number]
	at org.testng.asserts.SoftAssert.assertAll(SoftAssert.java:43)
	at TracksPageScripts.Linkedin_SignUp_PageScripts.fifthTestCase(Linkedin_SignUp_PageScripts.java:170)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:86)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:643)
	at org.testng.internal.Invoker.retryFailed(Invoker.java:967)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1146)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:129)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:112)
	at org.testng.TestRunner.privateRun(TestRunner.java:782)
	at org.testng.TestRunner.run(TestRunner.java:632)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:366)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:361)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:319)
	at org.testng.SuiteRunner.run(SuiteRunner.java:268)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1244)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1169)
	at org.testng.TestNG.run(TestNG.java:1064)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:114)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- fifthTestCase -->
        <test-method status="PASS" signature="takeScreenshort(org.testng.ITestResult)[pri:0, instance:TracksPageScripts.Linkedin_SignUp_PageScripts@3e57cd70]" name="takeScreenshort" is-config="true" duration-ms="909" started-at="2018-06-15T14:41:14Z" finished-at="2018-06-15T14:41:15Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[[TestResult name=fifthTestCase status=SKIP method=Linkedin_SignUp_PageScripts.fifthTestCase()[pri:5, instance:TracksPageScripts.Linkedin_SignUp_PageScripts@3e57cd70] output={null}]]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- takeScreenshort -->
        <test-method status="SKIP" signature="fifthTestCase()[pri:5, instance:TracksPageScripts.Linkedin_SignUp_PageScripts@3e57cd70]" name="fifthTestCase" duration-ms="3735" started-at="2018-06-15T14:41:15Z" finished-at="2018-06-15T14:41:19Z">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[The following asserts failed:
	expected [Please enter a valid email addressss] but found [Please enter a valid email address or mobile number]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: The following asserts failed:
	expected [Please enter a valid email addressss] but found [Please enter a valid email address or mobile number]
	at org.testng.asserts.SoftAssert.assertAll(SoftAssert.java:43)
	at TracksPageScripts.Linkedin_SignUp_PageScripts.fifthTestCase(Linkedin_SignUp_PageScripts.java:170)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:86)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:643)
	at org.testng.internal.Invoker.retryFailed(Invoker.java:967)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1146)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:129)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:112)
	at org.testng.TestRunner.privateRun(TestRunner.java:782)
	at org.testng.TestRunner.run(TestRunner.java:632)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:366)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:361)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:319)
	at org.testng.SuiteRunner.run(SuiteRunner.java:268)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1244)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1169)
	at org.testng.TestNG.run(TestNG.java:1064)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:114)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- fifthTestCase -->
        <test-method status="PASS" signature="takeScreenshort(org.testng.ITestResult)[pri:0, instance:TracksPageScripts.Linkedin_SignUp_PageScripts@3e57cd70]" name="takeScreenshort" is-config="true" duration-ms="888" started-at="2018-06-15T14:41:21Z" finished-at="2018-06-15T14:41:21Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[[TestResult name=fifthTestCase status=SKIP method=Linkedin_SignUp_PageScripts.fifthTestCase()[pri:5, instance:TracksPageScripts.Linkedin_SignUp_PageScripts@3e57cd70] output={null}]]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- takeScreenshort -->
        <test-method status="FAIL" signature="fifthTestCase()[pri:5, instance:TracksPageScripts.Linkedin_SignUp_PageScripts@3e57cd70]" name="fifthTestCase" duration-ms="3228" started-at="2018-06-15T14:41:21Z" finished-at="2018-06-15T14:41:25Z">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[The following asserts failed:
	expected [Please enter a valid email addressss] but found [Please enter a valid email address or mobile number]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: The following asserts failed:
	expected [Please enter a valid email addressss] but found [Please enter a valid email address or mobile number]
	at org.testng.asserts.SoftAssert.assertAll(SoftAssert.java:43)
	at TracksPageScripts.Linkedin_SignUp_PageScripts.fifthTestCase(Linkedin_SignUp_PageScripts.java:170)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:86)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:643)
	at org.testng.internal.Invoker.retryFailed(Invoker.java:967)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1146)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:129)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:112)
	at org.testng.TestRunner.privateRun(TestRunner.java:782)
	at org.testng.TestRunner.run(TestRunner.java:632)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:366)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:361)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:319)
	at org.testng.SuiteRunner.run(SuiteRunner.java:268)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1244)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1169)
	at org.testng.TestNG.run(TestNG.java:1064)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:114)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- fifthTestCase -->
        <test-method status="PASS" signature="takeScreenshort(org.testng.ITestResult)[pri:0, instance:TracksPageScripts.Linkedin_SignUp_PageScripts@3e57cd70]" name="takeScreenshort" is-config="true" duration-ms="912" started-at="2018-06-15T14:41:27Z" finished-at="2018-06-15T14:41:27Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[[TestResult name=fifthTestCase status=FAILURE method=Linkedin_SignUp_PageScripts.fifthTestCase()[pri:5, instance:TracksPageScripts.Linkedin_SignUp_PageScripts@3e57cd70] output={null}]]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- takeScreenshort -->
        <test-method status="PASS" signature="Teardown()[pri:0, instance:TracksPageScripts.Linkedin_SignUp_PageScripts@3e57cd70]" name="Teardown" is-config="true" duration-ms="2390" started-at="2018-06-15T14:41:27Z" finished-at="2018-06-15T14:41:30Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- Teardown -->
        <test-method status="PASS" signature="tearDown()[pri:0, instance:TracksPageScripts.Linkedin_SignUp_PageScripts@3e57cd70]" name="tearDown" is-config="true" duration-ms="362" started-at="2018-06-15T14:41:30Z" finished-at="2018-06-15T14:41:30Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDown -->
      </class> <!-- TracksPageScripts.Linkedin_SignUp_PageScripts -->
    </test> <!-- SignUp -->
  </suite> <!-- Automation Testing -->
</testng-results>
